src/PlusDataCollection/StealthLink/vtkPlusStealthLinkTracker.cxx

```diff
@@ -1018,7 +1018,7 @@ PlusStatus vtkPlusStealthLinkTracker::AreInstrumentPortNamesValid(bool& valid)
     }
     if (found == false)
     {
-      LOG_ERROR(toolIterator->second->GetPortName() << " has not been found in the server. Please make sure to use correct port names for the tools\n");
+      LOG_ERROR(toolIterator->second->GetPortName() << " has not been found in the StealthLink server. Please make sure that the desired tools have been added to the StealthStation, the correct port names are used in the config file, and that unused tools have been removed from the config file\n");

```

Comment:
I think something like this would be more clear:
... instrument is not available in connected StealthStation. Please make sure that the port name matches an instrument name in StealthStation or remove the corresponding DataSource element from DataSources and OutpuChannel elements in the config file